#goal definition
char[] inputData
char[] Endpoint
---
#result definition
char[] data
---
#feedback
int32[] sequence
